src/tasks/task-bholdus-bsc/services/bholdus-header-writer.service.ts

```diff
@@ -11,6 +14,15 @@ export class BholdusHeaderWriterService implements IService<BholdusBscTaskBus> {
 	constructor() {}
 
 	async start(): Promise<void> {
-		const api = await ApiPromise.create(getApiOptions());
+		// const api = await ApiPromise.create(getApiOptions());
+		const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+		const api = await ApiPromise.create({ provider: wsProvider });
+
+		// Do something
+		this.bus.channel(BholdusWriterMessage).subscribe({next:(message: BscPrimitivesBscHeader[]) =>{
+			let req = api.tx.bsc.verifyAndUpdateAuthoritySetSigned(message)
+			console.log("Result writer: ", req)
```

Comment:
Please log according to this rule:
`${Service Name} - log data` 

For example: `BholdusWriterService - submitting bsc headers to bsc pallet`